### ISSUE TITLE
Informal conference page needs to be split (508 audit finding)

### ISSUE THREAD

**Issue by luke-gcio** [department-of-veterans-affairs/va.gov-team#18049]
## User Story

As a user with issues processing pages with lots of content, I want to see the page in a way that does not confuse me.

## Description

In the Higher-Level Review informal conference page, after making an informal conference choice for yourself additional fields are revealed asking for your time availability. When your representative is selected, additional fields ask for your representatives name, phone number and time availability. These additional fields are all contained on the page and may lead to cognitive overload. Additionally, all form fields are contained within a single `fieldset` making the screenreader unable to discern the sections of the form.

## Acceptance Criteria
- [ ] Informal conference choice, representative information and time availability are split into 3 separate pages.
- [ ] Sign off by Josh, final audit, etc
